src/controller/command-handlers/signauthutils.hpp

```diff
@@ -45,6 +45,6 @@ extern template QString validateAndGetArgument<QString>(const QString& argName,
 extern template QByteArray
 validateAndGetArgument<QByteArray>(const QString& argName, const QVariantMap& args, bool allowNull);
 
-pcsc_cpp::byte_vector getPin(const electronic_id::ElectronicID& card, WebEidUI* window);
+void getPin(pcsc_cpp::byte_vector& pin, const electronic_id::ElectronicID& eid, WebEidUI* window);
```

Comment:
Why this is needed? Result is moved out from function

Reply:
Cleaner semantics: The physical memory where PIN resides is always owned by caller (and thus the copying problem simply does not exist)
AFAIK c++ std::move does not guarantee no-copy rule:
https://stackoverflow.com/questions/70078468/are-move-semantics-guaranteed-by-the-standard